pymatgen/analysis/elasticity/elastic.py

```diff
@@ -670,8 +674,10 @@ def omega(self, structure, n, u):
         """
         l0 = np.dot(np.sum(structure.lattice.matrix, axis=0), n)
         l0 *= 1e-10  # in A
-        weight = float(structure.composition.weight) * 1.66054e-27  # in kg
-        vol = structure.volume * 1e-30  # in m^3
+        # weight = float(structure.composition.weight) * 1.66054e-27  # in kg
+        weight = structure.composition.weight.to('kg').magnitude
+        # From Ayush: structure.volume is apparently not a Quantity, should it be?
+        vol = structure.volume * 1e-30  # in m^3 
         vel = (1e9 * self[0].einsum_sequence([n, u, n, u])
                / (weight / vol)) ** 0.5
         return vel / l0
```

Comment:
This is an example of how `CompatibleQuantity` can add value to pymatgen. Clearly there was a need here to do calcs in a specific set of units, and we currently rely on hard-coded conversions. Using `CompatibleQuantity`, this conversion is done automatically